pkgs/tools/networking/http2tcp/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, python3Packages
+, fetchurl
+}:
+
+python3Packages.buildPythonApplication rec {
+
+  pname = "http2tcp";
+  version = "0.5";
+
+  src = fetchurl {
+    url = "https://www.linta.de/~aehlig/http2tcp/${pname}-${version}.tar.gz";
+    sha256 = "34fb83c091689dee398ca80db76487e0c39abb17cef390d845ffd888009a5caa";
+  };
+
+  propagatedBuildInputs = with python3Packages; [ wsgitools ];
+
+  format = "other";
+
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir -p $out/{bin,share/${pname}}
+    cp http2tcp* $out/bin
+    cp Protocol $out/share/${pname}/
+  '';
```

Comment:
this doesn't look to be a normal python installation, you probably want to wrap the program with a python interpreter using python.withPackage https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#pythonwithpackages-function

Reply:
I have changed it that way